### PR TITLE
Add gene panel BRAIN to the master list

### PR DESCRIPTION
**Pull request by moahaegglund**
This PR adds gene panel BRAIN to the master list.

**How to prepare for test**:
- [x] install on stage/home/proj/production/servers/resources/hasta.scilifelab.se/update-cg-stage.sh add_gene_panel_to_master_list
- [x] activate stage: `us`

**How to test**:
- [x] run following command: `cg workflow mip-dna panel -p tidyjackal`

**Expected test outcome**:
- [x] cg should now print the included genepanels to the prompt. Gene panel "BRAIN" should be included in the list at the top.
- [x] Take a screenshot and attach or copy/paste the output.

**Review:**
- [x] code approved by @patrikgrenfeldt 
- [x] tests executed by @b4ckm4n 
- [x] "Merge and deploy" approved by @patrikgrenfeldt 
Thanks for filling in who performed the code review and the test!

This is a minor **version bump** because we add functionality in a backwards compatible manner

### CODE REVIEW

File: cg/meta/workflow/mip_dna.py [hunk 1]
```diff
@@ -33,6 +33,7 @@
     "NEURODEG",
     "mcarta",
     "Cardiology",
+    "BRAIN"
```
Comment (patrikgrenfeldt) [inline cg/meta/workflow/mip_dna.py]:
```suggestion
    "BRAIN",
```

**Comment by b4ckm4n** [comment]
Testing:

I don't think it worked as intended, missing version? Here is the output:

```[hiseq.clinical@hasta:~] [S_main] $ cg workflow mip-dna panel -p tidyjackal
Fetching gene panels ENDO from database
Fetching gene panels EP from database
Fetching gene panels IEM from database
Fetching gene panels IBMFS from database
Fetching gene panels mtDNA from database
Fetching gene panels MIT from database
Fetching gene panels PEDHEP from database
Fetching gene panels OMIM-AUTO from database
Fetching gene panels PIDCAD from database
Fetching gene panels PID from database
Fetching gene panels SKD from database
Fetching gene panels NMD from database
Fetching gene panels CTD from database
Fetching gene panels IF from database
Fetching gene panels NEURODEG from database
Fetching gene panels mcarta from database
Fetching gene panels Cardiology from database
Fetching gene panels BRAIN from database
No gene panel found
Panel BRAIN version None could not be found
Building hgncid_to_gene
##genome_build=37
```

**Comment by emiliaol** [comment]
It's probably because the BRAIN panel doesn't exist in Scout stage. I usually create a mock one with like one gene in it to test these kinds of things.

**Comment by b4ckm4n** [comment]
Works as intended! 🐸👍 
```
[daniel.backman@hasta:/home/proj/production/servers/resources/hasta.scilifelab.se] [S_main] master ± cg workflow mip-dna panel -p tidyjackal
Fetching gene panels ENDO from database
Fetching gene panels EP from database
Fetching gene panels IEM from database
Fetching gene panels IBMFS from database
Fetching gene panels mtDNA from database
Fetching gene panels MIT from database
Fetching gene panels PEDHEP from database
Fetching gene panels OMIM-AUTO from database
Fetching gene panels PIDCAD from database
Fetching gene panels PID from database
Fetching gene panels SKD from database
Fetching gene panels NMD from database
Fetching gene panels CTD from database
Fetching gene panels IF from database
Fetching gene panels NEURODEG from database
Fetching gene panels mcarta from database
Fetching gene panels Cardiology from database
Fetching gene panels BRAIN from database
Building hgncid_to_gene
```

**Comment by patrikgrenfeldt** [comment]
deployed to hasta and clinical-db

**Pull request merged**